Include/internal/pycore_compile.h

```diff
@@ -63,6 +64,115 @@ typedef struct {
     int u_firstlineno; /* the first lineno of the block */
 } _PyCompile_CodeUnitMetadata;
 
+struct _PyCompiler;
+
+typedef enum { OP_FAST, OP_GLOBAL, OP_DEREF, OP_NAME } _PyCompiler_optype;
```

Comment:
Should the enum members also be Py-prefixed? I feel logically they should be since they also create global names.